### PR TITLE
Run3-sim75 Remove namespace for comparison in SD classes

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Remove namespace for comparison in SD classes. DDD removes namespace while passing on the name to Geant4 while DD4Hep does not. So comparison of names  is better done after removing namespace.

#### PR validation:

Tested using workflows in the test areas of SimG4Core/Configuration/test

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
@bsunanda In the PR description, please provide some explanation for making this change.

File: SimG4CMS/Calo/plugins/EcalSimHitStudy.cc [hunk 1]
```diff
@@ -40,7 +41,7 @@
 #include <string>
 #include <vector>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
Comment (civanch) [inline SimG4CMS/Calo/plugins/EcalSimHitStudy.cc:44]: @bsunanda , you left DEBUG for purpose or forget?

Comment (bsunanda) [reply SimG4CMS/Calo/plugins/EcalSimHitStudy.cc:44]: I left it on purpose. I am working on a PR after this where I need to change this class as well. That will provide a true comparison between ddd and dd4hep. Could you sign this PR?

**Comment by bsunanda** [comment]
@cvuosalo Could you sign this PR? I have one more PR after this gets integrated

**Comment by bsunanda** [comment]
@cvuosalo @civanch Please approve this PR

**Comment by cvuosalo** [comment]
Do we know why there are 3509 comparison differences in the DD4hep workflows?

**Comment by bsunanda** [comment]
Earlier the material dependent corrections were wrong. So this is indeed a bug correction

**Comment by civanch** [comment]
@bsunanda , you left "#define EDM_ML_DEBUG" uncommented. Is it what should be?

**Comment by bsunanda** [comment]
I shall correct in the next PR. Please let this PR be merged
________________________________

**Comment by cvuosalo** [comment]
+1

**Comment by bsunanda** [comment]
@civanch I need your signature

**Comment by civanch** [comment]
+1

**Comment by silviodonato** [comment]
+1
@bsunanda please prepare a PR to close the edm_debug as soon as possible. We will build 11_2_0_pre10 tomorrow

**Comment by bsunanda** [comment]
@silviodonato Waiting for the IB with this PR to submit my next one

**Pull request merged**